vocab/credentials/v2/README.md

```diff
@@ -1,3 +1,52 @@
-# Updating vocabulary
+# Generate RDFS vocabulary files
 
-Vocabulary definitions are managed in `vocab.csv`. Add or change entries within this file. Regenerate `vocabulary.ttl`, `vocabulary.jsonld`, and `vocabulary.html` by running `mk_vocab.rb`.
+The script in the directory generates RDFS vocabulary files in JSON and Turtle formats, plus a human readable HTML file containing the vocabulary in RDFa, based on a simple vocabulary definition in a CSV file. Neither the script nor the CSV format is prepared for complex vocabularies; its primary goal is to simplify the generation of simple, straightforward RDFS vocabularies without, e.g., sophisticated OWL statements.
+
+## Definition of the vocabulary in the CSV file
+
+The vocabulary is defined in a CSV file, which contains the following columns: `category`, `id`, `property`, `value`, `label`, `upper value`, `domain`, `range`, `comment`, and `deprecated`. The `category` column defines the category of the row, and the interpretation of the row's content, i.e., of the other columns, depends on this category.
+
+The available categories, and the corresponding interpretation of the columns, are:
+
+- `vocab`: the prefix and the URL of the vocabulary that is being specified is provided in the `id` and `value` columns, respectively.
+- `prefix`: definition of a prefix, and corresponding URL, for each external external vocabulary in use, defined by the `id` and `value` columns, respectively. Note that some prefix/value pairs are defined by default, and it is not necessary to define them here; these are: `dc`, `owl`, `rdf`, `rdfs`, and `xsd`.
```

Comment:
I think the URLs of these predefined prefixes should be presented here, along with the prefixes themselves, to avoid any future misunderstanding or misinterpretation.

Reply:
Thanks @TallTed, done.